### ISSUE TITLE
Fix rpc_pbst test

### ISSUE THREAD

**Issue by akshaynexus** [ghost-coin/ghost-legacy#20]
Error log of the test:
```124/175 - rpc_psbt.py failed, Duration: 73 s

stdout:
2020-05-28T19:42:48.777000Z TestFramework (INFO): Initializing test directory /tmp/ghost_test_runner_P_🏃_20200528_222805/rpc_psbt_54
2020-05-28T19:43:59.687000Z TestFramework (INFO): PSBT spending unspendable outputs should have error message and Creator as next
2020-05-28T19:43:59.688000Z TestFramework (INFO): PSBT with invalid values should have error message and Creator as next
2020-05-28T19:43:59.688000Z TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/home/akshay/Documents/GitRepos/ghost-private/test/functional/test_framework/test_framework.py", line 114, in main
    self.run_test()
  File "/home/akshay/Documents/GitRepos/ghost-private/test/functional/rpc_psbt.py", line 457, in run_test
    assert_equal(analysis['next'], 'creator')
  File "/home/akshay/Documents/GitRepos/ghost-private/test/functional/test_framework/util.py", line 46, in assert_equal
    raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
AssertionError: not(updater == creator)
2020-05-28T19:43:59.740000Z TestFramework (INFO): Stopping nodes
2020-05-28T19:44:02.247000Z TestFramework (WARNING): Not cleaning up dir /tmp/ghost_test_runner_P_🏃_20200528_222805/rpc_psbt_54
2020-05-28T19:44:02.247000Z TestFramework (ERROR): Test failed. Test logging available at /tmp/ghost_test_runner_P_🏃_20200528_222805/rpc_psbt_54/test_framework.log
2020-05-28T19:44:02.248000Z TestFramework (ERROR): Hint: Call /home/akshay/Documents/GitRepos/ghost-private/test/functional/combine_logs.py '/tmp/ghost_test_runner_P_🏃_20200528_222805/rpc_psbt_54' to consolidate all logs


stderr:```

**Comment by oxarbitrage** [comment]
I think i found the problem of this one however i dont have a fix yet.

The test is failing after here: https://github.com/ghost-coin/ghost-private/blob/master/test/functional/rpc_psbt.py#L456

By decoding i can see:

```

./src/ghost-cli -regtest decodepsbt "cHNidP8BAHECAAAAAfA00BFgAm6tp86RowwH6BMImQNL5zXUcTT97XoLGz0BAAAAAAD/////AgD5ApUAAAAAFgAUKNw0x8HRctAgmvoevm4u1SbN7XL87QKVAAAAABYAFPck4gF7iL4NL4wtfRAKgQbghiTUAAAAAAABAR8AgIFq49AHABYAFJUDtxf2PHo641HEOBOAIvFMNTr2AAAA"
{
  "tx": {
    "txid": "1d7c2035b6abaccd04b43321a51ff541a9a7414aa7829e8cd72d302270922225",
...

      "witness_utxo": {
        "amount": 22000000.00000000,
        "scriptPubKey": {
...
```

The amount of `22000000.00000000` is invalid in bitcoin but as GHOST changed this at https://github.com/ghost-coin/ghost-private/blob/master/src/amount.h#L29 no error is thrown.

The fix will be to change the `22000000.00000000` for something like `82000000.00000000` however i am trying to figure out how to make the correct string with the change to submit a pr.

Any help is appreciated.

**Comment by Chesbob** [comment]
Patch in https://github.com/ghost-coin/ghost-private/pull/24
